packages/notifications/src/Notification/Notification.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useRef } from 'react';
+import { Alert, TextContent, Text, TextVariants, AlertActionCloseButton, AlertVariant } from '@patternfly/react-core';
+import { CloseIcon } from '@patternfly/react-icons';
+import './notification.scss';
+
+export interface NotificationProps {
+        /**
+     * Flag to automatically call `onDismiss` after `dismissDelay` runs out.
+     */
+         autoDismiss?: boolean,
+         /**
+          * Flag to show/hide notification close button.
+          */
+         dismissable?: boolean,
+         /**
+          * Funcation called after dismiss action is triggered. (id) => void
+          */
+         onDismiss: (id: number|string) => void,
+         /**
+          * Unique ID
+          */
+         id: number | string,
+         /**
+          * Alert variant. <a href="https://www.patternfly.org/v4/components/alert#types" target="_blank">More info.</a>
+          */
+         variant: AlertVariant,
```

Comment:
I don't think the variant is required prop. PF picks automatically `info` if no variant is passed.

Reply:
It's not required by PF but I think we should require it.